charts/generic/kubelet-service-kill/engine.yaml

```diff
@@ -18,8 +18,8 @@ spec:
         #   kubernetes.io/hostname: 'node02'        
           env:
             - name: TOTAL_CHAOS_DURATION
-              value: '90' # in seconds
+              value: '60' # in seconds
               
              # provide the target node name
             - name: TARGET_NODE
-              value: 'node-01'
+              value: ''
```

Comment:
I think this default value specifie a different node name than the one mentioned in node selector. Do we need to remove it?

Reply:
yes we can ask user to provide the node-name